README.md

```diff
@@ -14,6 +14,12 @@ The website is built using [mkdocs](https://www.mkdocs.org/). You can build the
 entire website and host it locally for testing your changes before submitting a
 pull request!
 
+Both of the following methods will start the mkdocs daemon to monitor for local changes to any of the files,
+automatically rebuild the site when a change is detected, and host all of it 
+locally at [http://127.0.0.1:8000](http://127.0.0.1:8000).
```

Comment:
I have moved this paragraph upwards.
It's not ideal as the last step is first now.

Moving it below is not a good solution as it's hard to tell where to start reading again.
Duplicating it is probably not good either.

Ideas?